### ISSUE TITLE
QoL: "Defeated X" from FFL2 would be a nice to declutter the combat text

### ISSUE THREAD

**Issue by SkankinGarbage** [psywombats/mgne#520]

**Comment by psywombats** [comment]
As in printing how many were defeated, vs a defeated message for each individual enemy?

**Comment by SkankinGarbage** [comment]
Yeah. I haven't noticed any situations where certain monsters in the group take less damage, and even if it's possible, you can't take any strategic advantage of it (like, say, I want to hit the 4th monster in the group). It would speed up combat a lot and make the combat log a lot tidier.
